config/default.yml

```diff
@@ -76,6 +76,18 @@ Betterment/UnsafeJob:
 Betterment/UnscopedFind:
   StyleGuide: '#bettermentunscopedfind'
 
+Betterment/RedirectStatus:
+  SafeAutoCorrect: false
```

Comment:
@rzane I noted `.rubocop_todo.yml` flags these cops as supporting safe autocorrection, although here is started otherwise, e.g.:

```
# your_app/.rubocop_todo.yml
# # This configuration was generated by
# `rubocop --auto-gen-config ....

# Offense count: 114
# This cop supports safe autocorrection (--autocorrect).
Betterment/RedirectStatus:
  Exclude:
    - 'app/.....'
```

Reply:
Oh, that's really strange. Any idea why that might be?

Reply:
Not yet, but I noted this happening to previous cops as well, ie `Betterment/HardcodedID`. 

Unsafe autocorrect (rubocop -A) addresses the cop though, it's just the misleading kind recorded in `.rubocop_todo.yml` 😕